functions/Get-DbaDbExtentDiff.ps1

```diff
@@ -0,0 +1,273 @@
+function Get-DbaDbExtentDiff {
+    <#
+        .SYNOPSIS
+            What percentage of a database has changed since the last full backup
+
+        .DESCRIPTION
+            This is only an implementation of the script created by Paul S. Randal to find what percentage of a database has changed since the last full backup
+            https://www.sqlskills.com/blogs/paul/new-script-how-much-of-the-database-has-changed-since-the-last-full-backup/
+
+        .PARAMETER SqlInstance
+            The target SQL Server instance
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+            $scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
+            To connect as a different Windows user, run PowerShell as that user.
+
+        .PARAMETER Database
+            The database where the script will be installed. Defaults to master
+
+        .PARAMETER WhatIf
+            Shows what would happen if the command were to run. No actions are actually performed.
+
+        .PARAMETER Confirm
+            Prompts you for confirmation before executing any changing operations within the command.
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Author: Viorel Ciucu, viorel.ciucu@gmail.com, cviorel.com
+
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            http://dbatools.io/Get-DbaDbExtentDiff
+
+        .EXAMPLE
+            Install the objects in master and msdb database
+            Get-DbaDbExtentDiff -SqlInstance RES14224
+
+        .EXAMPLE
+            Install the objects in [DBA] database
+            Get-DbaDbExtentDiff -SqlInstance RES14224 -Database DBA
+
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "High")]
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias('ServerInstance', 'SqlServer')]
+        [DbaInstance[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [object]$Database = "master",
+        [switch][Alias('Silent')]$EnableException
```

Comment:
since this is a new command, it doesn't need retro-compatibility with the old "Silent" alias. Use `[switch]$EnableException`